backend/canvas_app_explorer/data_class.py

```diff
@@ -0,0 +1,10 @@
+from dataclasses import dataclass
+
+@dataclass
+class ExternalTool:
+    label: str
+    id: int
+    is_hidden: bool
```

Comment:
I was thinking we may want to rename this at some point, something like `ExternalToolContext`. We already have a way of representing external tools with `models.LtiTool`, this just kinda extends it with some on information about navigation settings in a given context. I think I'm okay leaving it for now, but it may make sense soon as we try to combine this data with the `LtiTool` data.

Reply:
In Canvas they call this a "Tab" 

We could call it something like `CanvasTab` or `PlatformTab` if we wanted to be more generic. I'm fine renaming it afterward. I kind of like `PlatformTab`.